pkg/telemetryquerylanguage/functions/tqlotel/README.md

```diff
@@ -101,18 +101,24 @@ Examples:
 
 ## limit
 
-`limit(target, limit)`
+`limit(target, limit, priority_keys)`
 
 The `limit` function reduces the number of elements in a `pdata.Map` to be no greater than the limit.
 
-`target` is a path expression to a `pdata.Map` type field. `limit` is a non-negative integer. 
+`target` is a path expression to a `pdata.Map` type field. `limit` is a non-negative integer.
+`priority_keys` is a list of strings of attribute keys that won't be dropped during limiting.
+
```

Comment:
Is not a list is a variadic argument. I mean this means we cannot add arguments after and we don't need to use join/concat/etc.

Reply:
@TylerHelmuth how do we call these arguments?

Reply:
It is a limitation of the grammar at the moment. The grammar only supports 1 list parameter and it must be the last. Yes it is technically variadic, but I believe the docs currently call it a list.  There is open issue to add full list capabilities.